### ISSUE TITLE
[cli] cluster changes

### ISSUE THREAD

**Issue by dramich** [rancher/rancher#12313]
same table as UI for list
some marker on the "current" cluster and project
add options from the UI
add description flag for cluster create

**Comment by deniseschannon** [comment]
Available with v1.0.0-alpha11

**Comment by sangeethah** [comment]
Tested with v1.0.0-alpha11
1. Cluster list now includes following info with marker on the "current" cluster and project:

```
$ ./rancher cluster list 
CURRENT   ID              STATE     NAME       PROVIDER                    NODES     CPU       RAM            PODS
          cluster-6wrk9   active    santest3   Rancher Kubernetes Engine   1         0.54/1    0.14/0.87 GB   11/110
          cluster-b2qwt   active    santest2   Rancher Kubernetes Engine   1         0.53/1    0.12/0.87 GB   5/110
*         cluster-pbm4d   active    santest1   Rancher Kubernetes Engine   1         0.53/1    0.12/0.87 GB   6/110

```

2. We support create,add-node,delete, kubeconfig and import commands.

3. Description flag is available in cluster create command.
